airflow/providers/amazon/aws/hooks/ec2.py

```diff
@@ -33,21 +33,90 @@ class EC2Hook(AwsBaseHook):
         :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
     """
 
+    # Describe response
+    RESERVATIONS = 'Reservations'
+    INSTANCES = 'Instances'
+    STATE = 'State'
+    NAME = 'Name'
+    INSTANCE_ID = 'InstanceId'
```

Comment:
Please don't do this. Use the string directly. Having this extracted out just makes it harder to understand the code at the call site.

Reply:
Agreed Ash, but wanting to understand, is this an anti-pattern in python or against airflow coding convention, because I do see similar being used within the code base.
https://github.com/apache/airflow/blob/639972d995d848b16a3f283576efdbde28b8fdef/airflow/providers/google/cloud/hooks/dataflow.py#L93-L113

Reply:
Holding on this change unless must as this style is already used in the code base.

Reply:
@VijayantSoni in the example you posted those values are outside of the hook. Those are specific _types_.

I agree with Ash on using the strings directly (in your case) as those doesn't really have a category to specify them under, do they?

In the example the category is JobTypes and JobStatus.

Reply:
Thanks @feluelle for the clarification. got it now.